options.go

```diff
@@ -13,12 +13,23 @@ type Options struct {
 	// Print step definitions found and exit
 	ShowStepDefinitions bool
 
-	// Run scenarios in random order.
+	// Randomize causes scenarios to be run in random order.
 	//
-	// This is especially helpful for detecting situations
-	// where you have state leaking between scenarios, which
-	// can cause flickering or fragile tests.
-	RandomOrder bool
+	// Randomizing scenario order is especially helpful for detecting
+	// situations where you have state leaking between scenarios, which can
+	// cause flickering or fragile tests.
+	Randomize bool
```

Comment:
Hm, indeed. that becomes confusing. Go allows to create custom flag mappings, maybe it would be possible to leverage those with something like:

1. do not randomize order (most common case, with default nil value `0`),
`godog` without any options.
2. randomize order and let godog pick a pseudorandom seed (secondary common case),
`godog --randomize` could set `GODOG_SEED` env variable in order to be able to check or print what seed was used.
3. randomize order with a specific seed to try to reproduce a random failure (rare case).
`godog --randomize $(GODOG_SEED)`

Need to try to experiment in order to check whether that would be possible. Otherwise, maybe the way cucumber ruby has implemented it, would be more suitable